.github/workflows/esp32.yml

```diff
@@ -0,0 +1,77 @@
+name: ESP32
+on:
+  push:
+    branches: [master]
+    paths:
+      - '.github/workflows/esp32.yml'
+      - 'CMakeLists.txt'
+      - 'cmake/**'
+      - 'src/*'
+      - 'src/layer/*'
+      - 'src/layer/x86/**'
+      - 'src/layer/vulkan/**'
+      - 'tests/**'
```

Comment:
I don't think esp32 has any vulkan support atm
and esp32 is not x86, and there is not any tests built

delete these lines